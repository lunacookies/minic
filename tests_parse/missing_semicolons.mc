func main {
	var x = 50
	var y = x
	var z =
	var a = 1
	var b =
	set x = 92
}
