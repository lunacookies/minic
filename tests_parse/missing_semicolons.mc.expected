func main {
	var x = 50
	var y = x
	var z = <missing>
	var a = 1
	var b = <missing>
	set x = 92
}
tests_parse:23..24: error: missing “;”
tests_parse:34..35: error: missing “;”
tests_parse:43..44: error: missing expression
tests_parse:43..44: error: missing “;”
tests_parse:54..55: error: missing “;”
tests_parse:63..64: error: missing expression
tests_parse:63..64: error: missing “;”
tests_parse:75..76: error: missing “;”
